packages/lab4/src/main.sns.ts

```diff
@@ -0,0 +1,27 @@
+import * as AWS from 'aws-sdk';
+
+export const handler = async (event: AWSLambda.DynamoDBStreamEvent) => {
+  const sns = new AWS.SNS({apiVersion: '2010-03-31'});
+  const topicArn = process.env.TOPIC_ARN;
+
+  for (let record of event.Records) {
```

Comment:
Wäre es nicht etwas safer so? Bin aber nicht so der Profi.
```suggestion
  for (const record of event.Records) {
```

Reply:
🙏